src/encoded/schemas/consent.json

```diff
@@ -9,7 +9,7 @@
     "mixinProperties": [
         { "$ref": "mixins.json#/schema_version" },
         { "$ref": "mixins.json#/uuid" },
-        { "$ref": "mixins.json#/shared_status" },
+        { "$ref": "mixins.json#/standard_status" },
```

Comment:
@SenseCode Undo this status. Will be fixed in #206

Reply:
Closed #212 already!